lib/atp_scraper/activity.rb

```diff
@@ -53,6 +53,7 @@ def create_record(record, tournament)
         score: record[:score],
         win_loss: record[:win_loss],
         tournament_name: tournament[:name],
+        tournament_category: tournament[:category],
```

Comment:
__[reek]__
* [FeatureEnvy] refers to tournament more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)